libavpipe/src/avpipe_copy_mpegts.c

```diff
@@ -222,7 +218,8 @@ copy_mpegts_prepare_encoder(
 
     coderctx_t *encoder_context = &cp_ctx->encoder_ctx;
 
-    encoder_context->is_mpegts = decoder_context->is_mpegts;
+    encoder_context->live_proto = decoder_context->live_proto;
```

Comment:
Is setting `live_proto` here meaningful? It seems like an option that shouldn't mean anything on an encoder.